docs/operator-manual/notifications/index.md

```diff
@@ -45,3 +45,67 @@ So you can just use them instead of reinventing new ones.
     ```
 
 Try syncing an application to get notified when the sync is completed.
+
+## Namespace based configuration
+
+!!! important
+Available since v2.9
+
+A common installation method for Argo CD Notifications is to install it in a dedicated namespace to manage a whole cluster. In this case, the administrator is the only
+person who can configure notifications in that namespace generally. However, in some cases, it is required to allow end-users to configure notifications
+for their Argo CD applications. For example, the end-user can configure notifications for their Argo CD application in the namespace where they have access to and their Argo CD application is running in.
+
+This feature is based on applications in any namespace. See [applications in any namespace](../app-any-namespace.md) page for more information.
+
+In order to enable this feature, the Argo CD administrator must reconfigure the argocd-notification-controller workloads to add  `--application-namespaces` and `--self-service-notification-enabled` parameters to the container's startup command.
```

Comment:
does the notification SA require a `clusterole` vs a [role](https://github.com/argoproj/argo-cd/blob/ad4962352c7ac1aab66ec33dc2d81a150b80a927/manifests/base/notification/argocd-notifications-controller-role.yaml#L2)?

Reply:
Yes. This is documented in [app-any-namespace](https://github.com/argoproj/argo-cd/blob/e68d9ad2ad5566d2135bded67c9043fe4d14ea95/docs/operator-manual/app-any-namespace.md#adapt-kubernetes-rbac).